src/Microsoft.DotNet.XHarness.CLI/CommandArguments/WASM/WasmTestCommandArguments.cs

```diff
@@ -10,6 +10,28 @@ namespace Microsoft.DotNet.XHarness.CLI.CommandArguments.Wasm
 {
     internal class WasmTestCommandArguments : TestCommandArguments
     {
-        protected override OptionSet GetTestCommandOptions() => new OptionSet();
+        private string? _engine;
+        private string? _engineArgs;
+
+        public string Engine
+        {
+            get => _engine ?? throw new ArgumentException("Engine not specified");
+            set => _engine = value;
+        }
+
+        public string EngineArgs
+        {
+            get => _engineArgs ?? throw new ArgumentException("Engine args not specified");
+            set => _engineArgs = value;
+        }
+
+        protected override OptionSet GetTestCommandOptions() => new OptionSet{
+            { "engine=|e=", "Run on the listed javascript engine.",
+                v => Engine = v
+            },
+            { "arg=", "Argument to pass to the javascript engine",
+                v => EngineArgs = v
```

Comment:
I wonder if we could add this to be a list rather than a single list. In mono.options you can allow an argument to be passed more than once, for example:
```
xharness wasm test --arg="foo" --arg="bar" --arg="tar"
```
Then in the Mono.Options we can do something like:
```csharp
{ "arg=" , "Argument to pass to the javascript engine",
    v => EngineArgs.Add (v);
}
```
The process manager does have a method that takes a collection of arguments and will quote them for you, which will make your life much easier.